docs/en/developer.md

```diff
@@ -423,6 +423,41 @@ for the advanced settings.
 
 ## Advanced features
 
+### Allowing email login on LDAP login form
+
+If you have registered `LDAPAuthenticator` (see "Configure SilverStripe Authenticators" section), you may notice the only
+accepted login is username on the form. This may be confusing if your users are more familiar logging in with using their email
+address.
+
+To allow logging in with email in addition to username, set the following configuration:
+
+```yaml
+LDAPAuthenticator:
+  allow_email_login: 'yes'
+```
+
+Note that your LDAP users logging in must have the `mail` attribute set, otherwise this will not work.
```

Comment:
1) `logging`
2) I had to re-read this a couple of time until I understood that you meant the AD/Samba servers, not SilverStripe. How do I as an administrator/developer ensure that this attribute is set. Or is it automatic?